cmd/utils/utils.go

```diff
@@ -18,3 +19,11 @@ func DefaultPersistentPreRunE(cfgOpts config.ConfigOptions) func(_ *cobra.Comman
 		return nil
 	}
 }
+
+func SignatureClientResolver(signatureClientOpts *signing.SignatureClientOptions) (signing.SignatureClient, error) {
+	signatureClient, err := signing.NewSignatureClient(signatureClientOpts)
```

Comment:
I think it might make sense to just move the logic from `signing.NewSignatureClient` to this function, as that seems to be actually resolving the `singing.SignatureClient`. Wdyt?